### DISCUSSION TITLE
Add "product_type_id" to $filterable in Products

### DISCUSSION THREAD

**Discussion by justtomisfine** [lunarphp/lunar#1064] [Feature Requests]
- GetCandy version: 13.1
- Laravel Version: 8.83.14
- PHP Version: 8.1
- Database Driver & Version: Pgsql, 12.11

### Expected Behaviour:

Using `Product::search('query')->where('product_type_id', type_id)` would return a search filtered by product id.

### Actual Behaviour:

Get the following error:
```
Attribute `product_type_id` is not filterable. Available filterable attributes are: `status __soft_deleted brand skus`.
```

### Steps To Reproduce:

Try to filter a search with `where` method using `product_type_id`

#### Upgrade steps from 13.1 to X

Run `php artisan getcandy:meilisearch:setup`
This adds newly added `product_type_id` to index

(May have to run `php artisan getcandy:search:index` as well)

**Comment by alecritson** [comment]
Thanks for reporting, we're looking at making some improvements to search indexing, but if you need to add this in the meantime, you can tap into the filterable fields.

https://docs.getcandy.io/extending/search.html#overview

**Comment by justtomisfine** [comment]
Sounds great!
